booking-engine-api/operations/services.md

```diff
@@ -0,0 +1,290 @@
+# Services
+
+## Get services availability
+
+Get availability for the specified service for each time unit in the specified date interval.
+
+### Request
+
+`[ApiBaseUrl]/api/distributor/v1/services/getAvailability`
+
+```json
+{
+    "Client": "My Client 1.0.0",
+    "EnterpriseId": "3edbe1b4-6739-40b7-81b3-d369d9469c48",
+    "ServiceId": "40665467-5862-4932-a861-aa6b00e2b660",
+    "StartUtc": "202-01-01T00:00:00Z",
+    "EndUtc": "2022-01-31T00:00:00Z",
+    "CategoryIds": ["4037c0ec-a59d-43f1-9d97-d6c984764e8c"],
+    "FullAmounts": false,
+    "LanguageCode": "en-GB"
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Client` | string | required | Identification of the client as described in [Authorization](../guidelines/authorization.md). |
+| `EnterpriseId` | string | required | Unique identifier of the enterprise. |
+| `StartUtc` | string | required | Start date of the requested interval |
```

Comment:
It's a small detail, but some descriptions terminate in a full stop and some don't. I think it should be all or none.